x/staking/types/params_legacy.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+var (
+	KeyUnbondingTime     = []byte("UnbondingTime")
+	KeyMaxValidators     = []byte("MaxValidators")
+	KeyMaxEntries        = []byte("MaxEntries")
+	KeyBondDenom         = []byte("BondDenom")
+	KeyHistoricalEntries = []byte("HistoricalEntries")
+	KeyMinCommissionRate = []byte("MinCommissionRate")
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+// ParamTable for staking module
+// NOTE: Deprecated.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
```

Comment:
small question: is there a way to move this file into the migrations folder?

Reply:
I believe since you need to define `ParamSetPairs` in the `Params` type, it has to be local to the `types` package unfortunately. This can all be removed once x/params is completely deprecated in the next release.

Reply:
Ah I see, thanks.